server/services/mqtt/lib/handler/handleGladysMessage.js

```diff
@@ -0,0 +1,31 @@
+const { EVENTS } = require('../../../../utils/constants');
+const { devicestate } = require('./devicestate');
+const logger = require('../../../../utils/logger');
+
+/**
+ * @description Handle a new message receive in MQTT.
+ * @param {string} topic - MQTT topic.
+ * @param {Object} message - The message sent.
+ * @example
+ * handleGladysMessage(gladys.service.mqtt.client, 'gladys/device/create', '{ message: "content" }');
+ */
+function handleGladysMessage(topic, message) {
+  switch (topic) {
+    case 'gladys/master/device/create':
+      this.gladys.event.emit(EVENTS.DEVICE.NEW, JSON.parse(message));
+      break;
+    case 'gladys/master/state/create':
+      this.gladys.event.emit(EVENTS.DEVICE.NEW_STATE, JSON.parse(message));
+      break;
+    case 'gladys/master/devicestate/create':
```

Comment:
Used for backward compatibility.

Reply:
I don't think we want backward compatibility. We've done nothing in Gladys 4 to stay compatible with Gladys 3 (both are completely different), and to stay consistent I would avoid trying to be compatible at some place if we are not doing it everywhere.

And it's not working here, because a device/deviceState objects in Gladys 3 and Gladys 4 are completely different. Just throwing the object is not enough!

Reply:
I would suggest removing this part.

Reply:
Ok, so the comment on devicestate function is not relevant, I will remove this part and force people to rewrite their device code.